opendevin/server/listen.py

```diff
@@ -253,7 +253,6 @@ async def websocket_endpoint(websocket: WebSocket):
     """
     await websocket.accept()
 
-    session = None
```

Comment:
shouldn't this stay to reset it due to the potential return below?

Reply:
I believe it's pointless here.

Reply:
On second look, I did a search in the codebase and the method itself isn't referenced anywhere, unless I'm missing some automatic mechanism that picks it up? 🤔